### ISSUE TITLE
batchNormEvalMode is not supported anymore? What property should be used instead?

### ISSUE THREAD

**Issue by kfmn** [Microsoft/CNTK#369]
Hello! 

CNTK image recognition examples (such ad CIFAR-10, MNIST) use Batch Normalization and post-process trained network with a mel-script which contains a command like this:

SetPropertyForSubTree(CE, batchNormEvalMode, true)

But when using this command, I obtain exception: Invalid property, batchNormEvalMode, is not supported. 

How can I postprocess my net trained with batch normalization to make it work properly?

**Comment by Alexey-Kamenev** [comment]
Hello,
We have refactored batch norm code a bit and eliminated the need for using MEL to switch BN nodes from train mode to eval mode. All our samples should be updated accordingly, but please let us know if you still see something which uses MEL for BN.
Also, we added another time constant, `blendTimeConstant`, which allows to control how much of the smoothed history to "blend" into current minibatch statistics. We found this to be important on some workloads. For more info, please refer to the [documentation](https://github.com/Microsoft/CNTK/wiki/BatchNormalization) and samples/code.

**Comment by Alexey-Kamenev** [comment]
I'm closing the issue but feel free to re-open or create new one if you have any questions.
